.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+Issue: #
+<!-- Link to relevant issue. All PRs should be asociated with an issue -->
+
+## PR Type
+What kind of change does this PR introduce?
+
+<!-- Please check the one that applies to this PR using "x". -->
+```
+[ ] Bugfix
+[ ] Feature
+[ ] Code style update (formatting)
+[ ] Refactoring (no functional changes, no api changes)
+[ ] Build or CI related changes
+[ ] Documentation content changes
+[ ] Sample app changes
+[ ] Other... Please describe:
+```
+
+
+## What is the current behavior?
+<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->
+
+
+## PR Checklist
+Please check if your PR fulfills the following requirements:
+
```

Comment:
@nmetulev Is there a missing ` ``` `?

Reply:
This are checkboxes on purpose so the developer checks them when they've completed each one